### PR TITLE
check if remixd shared folder exists

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: libs/remixd/src/bin/remixd.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ function errorHandler (error: any, service: string) {
     console.log('\x1b[33m%s\x1b[0m', '[WARN] You may now only use IDE at ' + program.remixIde + ' to connect to that instance')
   }
 
-  if (program.sharedFolder) {
+  if (program.sharedFolder && existsSync(absolutePath('./', program.sharedFolder))) {
```
Comment (yann300) [inline libs/remixd/src/bin/remixd.ts:86]: `sharedFolder` can be either absolute path or relative path, is this test taking this in account?

File: libs/remixd/src/bin/remixd.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ function errorHandler (error: any, service: string) {
       })
       // Run hardhat service if a hardhat project is shared as folder
       const hardhatConfigFilePath = absolutePath('./', program.sharedFolder) + '/hardhat.config.js'
```
Comment (ioedeveloper) [inline libs/remixd/src/bin/remixd.ts:104]: Is hardhat path limited to relative path?

File: libs/remixd/src/bin/remixd.ts (hunk 1 above)
Comment (bunsenstraat) [reply libs/remixd/src/bin/remixd.ts:86]: I have tested this and it seems fine to me

**Review by bunsenstraat — APPROVED** [review]
I ran some tests with it, relative, absolute. seems fine to me.

**Comment by Aniket-Engg** [comment]
> I ran some tests with it, relative, absolute. seems fine to me.

Cool Thanks

**Pull request merged**